R/verify_avaln.R

```diff
@@ -30,15 +30,15 @@ verify_avaln<-function(data,id_col="STUDYID",AVALC_col="AVALC",AVALN_col="AVALN"
   check_AVALN <-
     data %>%
     select(.data[[id_col]],.data[[param_col]],.data[[AVALC_col]],.data[[AVALN_col]]) %>% 
+    filter(is.na(.data[[AVALN_col]])|.data[[AVALN_col]]=="")%>%
     group_by(.data[[id_col]],.data[[param_col]],.data[[AVALC_col]],.data[[AVALN_col]]) %>%
     distinct() %>%
-    ungroup() %>% 
-    rowwise() %>% 
     mutate(check=case_when(
       .data[[AVALC_col]] ==.data[[AVALN_col]] ~"TRUE",
       is.na(.data[[AVALC_col]])%in% is.na(.data[[AVALN_col]])~ "TRUE",
```

Comment:
I think that you mean for this to be `==` rather than `%in%`.  `%in%` will probably always be true, so it may accidentally skip the false trigger below.

Reply:
Yes with rowwise() removed now, == should be the correct operator to use here.